### PR TITLE
ENG-90 remove local PageTreeSelector file and use npm package instead

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

File: src/ui/pages/common/PageTreeSelectorContainer.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import { connect } from 'react-redux';
 
-import PageTreeSelector from 'ui/pages/common/PageTreeSelector';
+import { PageTreeSelector } from '@entando/pagetreeselector';
 
 import { handleExpandPage } from 'state/pages/actions';
 
 import { getPageTreePages } from 'state/pages/selectors';
 
+import '@entando/pagetreeselector/dist/css/index.css';
```
Comment (gidesan) [inline src/ui/pages/common/PageTreeSelectorContainer.js]: Current practice in appbuilder is importing css in appbuilder's index.js (see stylesheets import of @entando/menu and @entando/pages for reference).

@nicpuddu what do you think?

**Comment by entando-service** [comment]
All is well

File: src/ui/pages/common/PageTreeSelectorContainer.js (hunk 1 above)
Comment (ichalagashvili) [reply src/ui/pages/common/PageTreeSelectorContainer.js]: np, I will move it there

**Comment by entando-service** [comment]
All is well

**Pull request merged**